File: Data Structures & Algorithms/duplicate-integer/submission-1.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        unordered_set<int> _set;
        for(int i=0; i<nums.size(); i++){
            if(_set.contains(nums[i])){
                return true;
            }
            _set.insert(nums[i]);
        }
        return false;
    }
};

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-1.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i, j = 0, 1
        while j<len(nums):
            if nums[i] == nums[j]:
                #equal
                j+=1
            else:
                #unequal
                i+=1
                nums[i] = nums[j]
        return i+1
